wolvic/browser/vr/wvr_manager.cc

```diff
@@ -684,6 +735,21 @@ void WvrManager::UpdateLayerBounds(int16_t frame_index,
                                    const gfx::RectF& left_bounds,
                                    const gfx::RectF& right_bounds,
                                    const gfx::Size& source_size) {
+  if (!ValidateRect(left_bounds) || !ValidateRect(right_bounds)) {
```

Comment:
How is it possible we get RectF with nan? Where are we getting this from?

Reply:
As the name suggests, `RectF` values are floating and may result in NaN depending the result of operations. Regardless, I have copied _ipsis litteris_ this snip from:
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/android/vr/gvr_scheduler_delegate.cc;l=71?q=ValidateRect

I'm seeing a similar usage of that for Wayland, for instance: 
https://source.chromium.org/chromium/chromium/src/+/main:ui/ozone/platform/wayland/host/wayland_frame_manager.cc?q=ValidateRect


Are you suggesting me to change something, sorry Dape?

Reply:
No, no. Just making sure we do that check as early as we can. Is that the case?

Reply:
Yes.